unic/ucd/tests/category_tests.rs

```diff
@@ -18,8 +18,112 @@ extern crate unic_utils;
 
 use unic_ucd::bidi::BidiClass;
 use unic_ucd::normal::is_combining_mark;
+use unic_ucd::category::GeneralCategory as GC;
 use unic_utils::iter_all_chars;
 
+/// `normal::is_combining_mark` and `GeneralCategory::is_mark()` are expected to return
+/// the same results.
+#[test]
+fn test_gen_cat_against_normal() {
```

Comment:
After #57, this may be not testing the actual `ucd-normal` table for `is_combining_mark` but be calling on the implementation that calls back into `ucd-category`, because [I set `unic-ucd` to turn on the normal feature `unic-ucd-category`](https://github.com/behnam/rust-unic/pull/57/files#diff-588a4bcdb9c391c05ffe2270763cef14R22).

This test will need to either work around that or live instead in unic-normal. @behnam?